tests/external_pg_test.go

```diff
@@ -0,0 +1,68 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/bytebase/bytebase/common"
+	"github.com/bytebase/bytebase/resources/postgres"
+	"github.com/stretchr/testify/require"
+)
+
+type fakeExternalPg struct {
```

Comment:
We don't have to introduce this struct if only used for a single test.

Reply:
This can be used as an outline for external PostgreSQL, we may add other tests later.